pywwt/layers.py

```diff
@@ -502,6 +601,62 @@ def _on_cmap_vmin_vmax_change(self, *value):
         self.parent._send_msg(event='table_layer_set', id=self.id,
                               setting='colorMapColumn', value=CMAP_COLUMN_NAME)
 
+
+    @observe('time_att')
+    def _on_time_att_change(self, *value):
+        # Convert time column to UTC so WWT displays points at expected times
+        ### ISSUE: Currently not robust to changes in DST.
+        # Needs GMT instead of UTC?
+
+        # The conversion only seems needed in Qt, so check the widget version
+        if type(self.parent).__name__.find('Qt') < 0:
```

Comment:
What versions is this actually checking?

Reply:
I'm trying to check whether we're in an instance of `WWTJupyterWidget` or `WWTQtClient` since Chrome/Firefox and WebKit handle `Date` objects differently. Does this not work for you?

Ideally, this step won't be necessary after we iron out the current issues with time zones.